python/iceberg/core/base_metastore_tables.py

```diff
@@ -50,4 +50,4 @@ def default_warehouse_location(self, conf, database, table):
         if warehouse_location is None:
             raise RuntimeError("Warehouse location is not set: hive.metastore.warehouse.dir=null")
 
-        return "%s/%s.db/%s".format(warehouse_location, database, table)
+        return f"{warehouse_location}/{database}.db/{table}"
```

Comment:
Was this required by Mypy? Looks more like a style change.

Reply:
The thing is actually not templated:
```
MacBook-Pro-van-Fokko:incubator-iceberg fokkodriesprong$ python3
Python 3.6.6 (v3.6.6:4cf1f54eb7, Jun 26 2018, 19:50:54) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> '%s'.format('vo')
'%s'
>>> '{}'.format('vo')
'vo'
```